src/Commands/Backup/InfoCommand.php

```diff
@@ -31,14 +31,16 @@ class InfoCommand extends SingleBackupCommand
      *
      * @param string $site_env Site & environment in the format `site-name.env`
      * @option string $file [filename.tgz] Name of backup file
-     * @option string $element [all|code|files|database|db] Backup element to retrieve
+     * @option string $element [code|files|database|db] Backup element to retrieve
      *
      * @usage <site>.<env> Displays information about the most recent backup of any type in <site>'s <env> environment.
      * @usage <site>.<env> --file=<file_name> Displays information about the backup with the file name <file_name> in <site>'s <env> environment.
      * @usage <site>.<env> --element=<element> Displays information about the most recent <element> backup in <site>'s <env> environment.
      */
-    public function info($site_env, array $options = ['file' => null, 'element' => 'all',])
+    public function info($site_env, array $options = ['file' => null, 'element' => 'files',])
```

Comment:
Why is `all` being removed here?

Reply:
Because it's inconsistent and actually has never worked (it used to print only the last backup item)